AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/Storage.smithy

```diff
@@ -233,15 +330,15 @@ in the terminal state of a Mutation,
 a page of version (decrypt only) items,
 conditioned on:
 - every version already exsisting
-- the original of a Mutation Lock commits to the original provided  
-- the terminal of a Mutation Lock commits to the terminal provided  
+- every version's enc has not changed
```

Comment:
```suggestion
- every version's enc field has not changed
```

Reply:
How about:
```suggestion
- every version's cipher-text had not changed
```

This clarifies what the `enc` field holds.
Technically, 
if a custom storage implementation did not use a database but something like S3,
"fields" may not exist.